scripts/closure-compiler/build-devapp-bundle.sh

```diff
@@ -0,0 +1,91 @@
+#!/bin/bash
+
+# Script that bundles the dev-app using the Google Closure compiler.
+# This is script is used to verify closure-compatiblity of all Material components.
```

Comment:
I'd like to run Closure Compiler over the e2e-app and run the e2e tests over that in order to ensure that everything is working; I think this PR is a good first step to ensure that that the code at least _passes_.